### PR TITLE
fix: show version on eas-update for production release

### LINKED ISSUE #830: Fix no-message when calling eas-update for a production release
It would make sense to provide the release version instead

### PR DESCRIPTION
**Pull request by brunopascottini**
Closes #830

### CODE REVIEW

**Review by simoneb — COMMENTED** [review]
shall we consider removing the default value of this input in the called workflow?

**Comment by brunopascottini** [comment]
@simoneb Sounds good, I'll also remove from the other input and from the CI workflow.

File: .github/workflows/ci.yml
Comment (simoneb) [inline .github/workflows/ci.yml:38]: why did you remove this?

File: package.json
Comment (simoneb) [inline package.json:56]: why did  you update package versions?

Comment (brunopascottini) [reply package.json:56]: Build was failing on CI and I thought it would be better to fix now than leave it to dependabot.

File: .github/workflows/ci.yml
Comment (brunopascottini) [reply .github/workflows/ci.yml:38]: I checked the previous commits that triggered this workflow and this default message was not being used.

File: package.json
Comment (simoneb) [reply package.json:56]: oh interesting. it should have been caught before being merged to master, that's weird!

**Review by simoneb — APPROVED** [review]

**Pull request merged**
